src/g_game.c

```diff
@@ -2186,7 +2186,7 @@ void G_Ticker(void)
   if (demoplayback && sendsave)
     {
       sendsave = false;
-      G_DoSaveGame();
+      gameaction = ga_savegame;
```

Comment:
This will postpone game saving by one tic. Is this considered safe?

Reply:
I think so, but I'm not sure. This is how a normal non-demo save works.